### PR TITLE
Fix documentation for Docker API filters

### PR DESCRIPTION
**Pull request by xperimental**
Fixes the error in the documentation found during #10081

### CODE REVIEW

File: docs/configuration/configuration.md [hunk 1]
```diff
@@ -868,7 +866,9 @@ role: <string>
 # Optional filters to limit the discovery process to a subset of available
 # resources.
 # The available filters are listed in the upstream documentation:
-# https://docs.docker.com/engine/api/v1.40/#operation/ContainerList
+# Services: https://docs.docker.com/engine/api/v1.40/#operation/ServiceList
+#    Tasks: https://docs.docker.com/engine/api/v1.40/#operation/TaskList
+#    Nodes: https://docs.docker.com/engine/api/v1.40/#operation/NodeList
```
Comment (LeviHarrison) [inline docs/configuration/configuration.md:871]: Is the indentation intentional here?

**Review by LeviHarrison — COMMENTED** [review]
Thanks so much for this. One small thing.

File: docs/configuration/configuration.md (hunk 1 above)
Comment (xperimental) [reply docs/configuration/configuration.md:871]: Yes, I thought it looked nicer when the links aligned properly. It's personal preference, I can undo the alignment if you like.

Comment (roidelapluie) [reply docs/configuration/configuration.md:871]: I would undo it because it looks like tasks and nodes depend on the first link. Thanks.

Comment (xperimental) [reply docs/configuration/configuration.md]: Sure, I have reverted the indentation change. Should I squash the commits or are you doing that anyway during merge?

**Review by LeviHarrison — APPROVED** [review]
Thanks!

**Comment by LeviHarrison** [comment]
Closing/re-opening to trigger CI

**Comment by xperimental** [comment]
If you like I can just amend the last commit and re-push to trigger the CI again, if that would help.

**Pull request merged**